modules/UI/videolayout/RemoteVideo.js

```diff
@@ -245,6 +245,10 @@ RemoteVideo.prototype._requestRemoteControlPermissions = function () {
             {user: this.user.getDisplayName()
                 || interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME}
         );
+        let pinnedId = this.VideoLayout.getPinnedId();
```

Comment:
I changed it to pin the controlled participant only if permissions have been granted (the remote control session start).